hack/Dockerfile

```diff
@@ -13,8 +13,17 @@
 # limitations under the License.
 
 FROM golang:GO_VERSION
+ARG UID=0
```

Comment:
ARG is new to me -- is there a convention to document the arguments to a dockerfile in a comment?